doc/howto/container_gpu_passthrough_with_docker.md

```diff
@@ -0,0 +1,155 @@
+
+        #cloud-config
+        package_update: true
+        packages:
+          - docker.io
+        write_files:
+          - path: /etc/docker/daemon.json
+            permissions: "0644"
+            owner: root:root
+            content: |
+            {
+                "runtimes": {
+                  "nvidia": {
+                    "args": [],
+                    "path": "nvidia-container-runtime"
+                  }
+                }
+            }
+          - path: /root/run_tensorrt.sh
+            permissions: "0755"
+            owner: root:root
+            content: |
+            #!/bin/bash
+            echo "OS release,Kernel version"
+            (. /etc/os-release; echo "${PRETTY_NAME}"; uname -r) | paste -s -d,
+            echo
+            nvidia-smi -q
+            echo
+            exec bash -o pipefail -c "
+            cd /workspace/tensorrt/samples
+            make -j4
+            cd /workspace/tensorrt/bin
+            ./sample_onnx_mnist
+            retstatus=\${PIPESTATUS[0]}
+            echo \"Test exited with status code: \${retstatus}\" >&2
+            exit \${retstatus}
+            "
+        runcmd:
+          - systemctl start docker
+          - systemctl enable docker
+          - usermod -aG docker root
+          - curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg
+            --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg
+          - curl -s -L
+            https://nvidia.github.io/libnvidia-container/stable/deb/nvidia-container-toolkit.list
+            | sed 's#deb https://#deb
+            [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg]
+            https://#g' | tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
+          - sed -i -e '/experimental/ s/^#//g'
+            /etc/apt/sources.list.d/nvidia-container-toolkit.list
+          - apt-get update
+          - DEBIAN_FRONTEND=noninteractive apt-get install -y nvidia-container-toolkit
+          - nvidia-ctk runtime configure --runtime=docker
+            --config=/etc/docker/daemon.json
+          - systemctl restart docker
+
+1. Apply this `cloud-init` setup to your instance:
+
+        lxc config set t1 cloud-init.user-data - < cloud-init.yml
+
+1. Start the instance:
+
+        lxc start t1
+
+1. Wait for the `cloud-init` process to finish:
+
+        lxc exec t1 -- cloud-init status --wait
+
+1. Once cloud-init is finished, we can shell into the instance:
+
+        lxc shell t1
+
+1. We now need to edit the NVIDIA container runtime. Ensure that we have set `no-cgroups = true` in `/etc/nvidia-container-runtime/config.toml`. Once modified, make sure to restart the NVIDIA container runtime for Docker with `nvidia-ctk runtime configure --runtime=docker --config=/etc/docker/daemon.json && systemctl restart docker`
```

Comment:
```suggestion
1. Edit the NVIDIA container runtime. 
   Open `/etc/nvidia-container-runtime/config.toml` and set `no-cgroups = true`.
   Once modified, restart the NVIDIA container runtime for Docker with the following command:
   
       nvidia-ctk runtime configure --runtime=docker --config=/etc/docker/daemon.json && systemctl restart docker
```

Reply:
"Ensure that we have set" - we haven't done that, or did I miss it in any of the previous steps?

Reply:
fixed